README.md

```diff
@@ -97,7 +97,12 @@ and then it is possible to include the artifacts as dependencies to a `pom.xml`:
 ## Contributing
 
 You can contact the Truffle developers at graal-dev@openjdk.java.net mailing
-list.
+list. To contribute a change, verify it using
+```bash
+$ mx gate
+```
+and start a [pull request](https://help.github.com/articles/using-pull-requests/).
+Detailed info can be found in [contributing document](CONTRIBUTING.md).
```

Comment:
Oops - missed that. Github review don't make the boundaries between files clear enough in my opinion.

In this case, there should be a "the" after "found in".